src/Resources/app/storefront/src/storefront/js/glightbox.js

```diff
@@ -0,0 +1,2573 @@
+(function (global, factory) {
```

Comment:
Can't we install it with npm instead of putting 3rd party source code into our repository? 
`npm install glightbox`